packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -172,29 +172,28 @@ describe('PerformanceMonitoring', function() {
   })
 
   it('should calculate browser responsiveness', function() {
-    var tr = new Transaction('transaction', 'transaction', {})
-    tr.end()
-
-    tr._start = 1
+    const tr = new Transaction('transaction', 'transaction', {
+      startTime: 1
+    })
+    tr.end(400)
 
-    tr._end = 400
     tr.browserResponsivenessCounter = 0
-    var resp = performanceMonitoring.checkBrowserResponsiveness(tr, 500, 2)
+    var resp = performanceMonitoring.checkBrowserResponsiveness(tr)
     expect(resp).toBe(true)
 
     tr._end = 1001
     tr.browserResponsivenessCounter = 2
-    resp = performanceMonitoring.checkBrowserResponsiveness(tr, 500, 2)
+    resp = performanceMonitoring.checkBrowserResponsiveness(tr)
     expect(resp).toBe(true)
 
     tr._end = 1601
     tr.browserResponsivenessCounter = 2
-    resp = performanceMonitoring.checkBrowserResponsiveness(tr, 500, 2)
+    resp = performanceMonitoring.checkBrowserResponsiveness(tr)
     expect(resp).toBe(true)
 
-    tr._end = 3001
+    tr._end = 3501
```

Comment:
If we didn't change the logic for `checkBrowserResponsiveness` this value should not be changed then.